target/hwaccess/hwBaseAccess.C

```diff
@@ -0,0 +1,26 @@
+#include "hwBaseAccess.H"
+
+namespace hwaccess
+{
+int HwBaseAccess::getCfam(TargetPtr_t target, uint32_t addr, uint32_t& val)
+{
+    if(target == nullptr)
+    {
+        return -1;
```

Comment:
It would be better to have debug traces on why we are returning an error. Similar comment for all the places where we return with an error.